tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+# role tasks
+- name: Upgrade kexec-tools to latest
+  yum:
```

Comment:
Change this to "package" so it will work going forward into future systems that don't use `yum`.